cmd/logout.go

```diff
@@ -0,0 +1,27 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var componentLogoutCmd = &cobra.Command{
+	Use:   "logout",
+	Short: "Log out of the active session",
+	Long:  "Log out of the active session",
+	Example: `  # Logout
+  odo logout
+	`,
+	Args: cobra.NoArgs,
+	Run: func(cmd *cobra.Command, args []string) {
+		client := getOcClient()
+		err := client.RunLogout()
+		checkError(err, "")
+	},
+}
+
+func init() {
+	// Add a defined annotation in order to appear in the help menu
+	componentListCmd.Annotations = map[string]string{"command": "utility"}
+
+	rootCmd.AddCommand(componentLogoutCmd)
```

Comment:
You're missing these commands: https://github.com/redhat-developer/odo/blob/master/cmd/log.go#L57-L59

Reply:
ack, fixed